routers/api/v1/misc/markdown.go

```diff
@@ -65,20 +64,20 @@ func Markdown(ctx *context.APIContext, form api.MarkdownOption) {
 		if form.Wiki {
 			_, err := ctx.Write([]byte(markdown.RenderWiki(md, urlPrefix, meta)))
 			if err != nil {
-				ctx.Error(http.StatusInternalServerError, "", err)
+				ctx.Error(500, "", err)
```

Comment:
Why change this?

Reply:
code formation either use `http.Status`* or numbers to be more strict

Reply:
I would say use http.Status*